traverson4j-hc5/build.gradle

```diff
@@ -1,6 +1,6 @@
 dependencies {
-    compile 'org.apache.httpcomponents:httpclient:4.5.6',
-            'org.apache.httpcomponents:httpmime:4.5.6',
+    compile 'org.apache.httpcomponents.client5:httpclient5:5.0.3',
+            'org.apache.httpcomponents:httpmime:4.5.13',
```

Comment:
How come this is v4? I think this brings in V4 and V5 of the http client?

Reply:
There is no v5 equivalent. Will see if we can remove it entirely

Reply:
removed